src/components/Activity/ActivityItem.js

```diff
@@ -60,37 +62,28 @@ const ActivityItem = ({ activity, onClose }) => {
       <div
         css={`
           grid-area: time;
-          opacity: 0.7;
-          font-size: 12px;
-          font-weight: 600;
-          color: #6d777b;
-          letter-spacing: 0;
-          text-align: right;
-          line-height: 16px;
-          white-space: nowrap;
-          & > span {
-            vertical-align: sub;
-          }
+          justify-self: end;
         `}
       >
-        <TimeTag date={activity.createdAt} style={{ marginRight: 10 }} />
+        <TimeTag date={activity.createdAt} />
       </div>
       <div
         css={`
           grid-area: content;
           overflow: hidden;
           position: relative;
-          margin-top: 10px;
-          margin-bottom: 0;
-          line-height: 22px;
+          margin: 10px 0 0;
           font-size: 15px;
-          color: #000000;
-          letter-spacing: 0;
-          line-height: 22px;
         `}
       >
         <ItemContent text={activity.description} />
         <StatusMessage activity={activity} />
+        <TransactionProgress
+          status={activity.status}
+          createdAt={activity.createdAt}
+          minedAtEstimate={-1}
```

Comment:
No need to add this prop no? `-1` is already the default

Reply:
Oh right, the original idea was to get it from an API instead of an arbitrary value, but I couldn’t find any API giving this information. Let’s remove it from now yes :+1:

Reply:
Did you remove the default value and removed this prop?

Reply:
Yes I removed the prop, and also the default value in the function parameters, since it is already present in `defaultProps`.